src/CSnakes.Runtime/Python/PyObject.cs

```diff
@@ -304,6 +304,49 @@ private static bool Compare(PyObject left, PyObject right, CPythonAPI.RichCompar
         }
     }
 
+    /// <summary>
+    /// Check if the object is false. Equivalent to the <c>not</c> operator in Python.
+    /// </summary>
+    public static bool operator !(PyObject obj)
+    {
+        if (obj.Is(False))
+            return true;
+
```

Comment:
Add a shortcut condition for None as well

Reply:
Good point and done with 0ccbfe30e57797ca6a84c27a52418b11e71aca30.